### PR TITLE
Hotfix disabled persister on trie sync

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

**Review by SebastianMarian — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

File: process/block/preprocess/transactionsV2.go [hunk 1]
```diff
@@ -227,7 +227,7 @@ func (txs *transactions) processTransaction(
 			"senderUsername", tx.GetSndUserName(),
 			"receiverAddr", tx.GetRcvAddr(),
 			"receiverUsername", tx.GetRcvUserName(),
-			"data", string(tx.Data),
+			"data", tx.Data,
```
Comment (bogdan-rosianu) [inline process/block/preprocess/transactionsV2.go:230]: please revert the changes for the data field. it is directly convertible to string and the logger will display them as hex, which implies a further step when debugging

Comment (bogdan-rosianu) [reply process/block/preprocess/transactionsV2.go:230]: applicable for the entire changeset of this commit

File: epochStart/bootstrap/startInEpochScheduled.go [hunk 2]
```diff
@@ -288,9 +288,9 @@ func (ses *startInEpochWithScheduledDataSyncer) filterScheduledIntermediateTxs(
 			log.Debug("startInEpochWithScheduledDataSyncer.filterScheduledIntermediateTxs",
 				"intermediate tx hash", txHash,
 				"intermediate tx nonce", txHandler.GetNonce(),
-				"intermediate tx sender address", string(txHandler.GetSndAddr()),
-				"intermediate tx receiver address", string(txHandler.GetRcvAddr()),
-				"intermediate tx data", string(txHandler.GetData()),
+				"intermediate tx sender address", txHandler.GetSndAddr(),
+				"intermediate tx receiver address", txHandler.GetRcvAddr(),
+				"intermediate tx data", txHandler.GetData(),
```
Comment (bogdan-rosianu) [inline epochStart/bootstrap/startInEpochScheduled.go:293]: revert for the data field

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by SebastianMarian — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**